subsys/bluetooth/shell/iso.c

```diff
@@ -417,7 +471,7 @@ static int cmd_big_term(const struct shell *shell, size_t argc, char *argv[])
 #endif /* CONFIG_BT_ISO_BROADCAST */
 
 SHELL_STATIC_SUBCMD_SET_CREATE(iso_cmds,
-	SHELL_CMD_ARG(bind, NULL, "[dir] [interval] [packing] [framing] "
+	SHELL_CMD_ARG(bind, NULL, "[dir=<,>,<>] [interval] [packing] [framing] "
```

Comment:
```suggestion
	SHELL_CMD_ARG(bind, NULL, "[dir=tx,rx,txrx] [interval] [packing] [framing] "
```

Did some merge conflict go wrong here?

Reply:
Yep, this might be a fixup to the wrong patch so it end up not fixing it in the correct place, I will fix that.